src/scry/completion/dependency_graph.cr

```diff
@@ -74,13 +78,12 @@ module Scry::Completion::DependencyGraph
         .flat_map { |d| Dir.glob(d) }
         .each { |file| process_requires(file, graph) }
 
-      prelude_node = graph[/src\/prelude.cr$/]
-      Log.logger.debug("Finished building the dependancy graph got these nodes:#{graph.each.to_a.map(&.first)}")
-      return graph if prelude_node.nil?
-
-      graph.each.reject { |e| e == prelude_node.not_nil!.value }.each do |key, _|
-        graph[key].connections << prelude_node.not_nil!
+      if prelude_node = graph[/src\/prelude.cr$/]
+        graph.each.reject { |e| e == prelude_node.value }.each do |key, _|
+          graph[key].connections << prelude_node
+        end
       end
+      Log.logger.debug("Finished building the dependancy graph got these nodes:#{graph.each.to_a.map(&.first)}")
       graph
```

Comment:
I removed `.not_nil!` here as well :wink:

Reply:
Oh, looks like this doesn't work for some reason, let me revert it :sweat_smile: